templates/advantage/maintenance.html

```diff
@@ -1,23 +1,25 @@
 {% extends "templates/one-column.html" %}
-{% block title %}Ubuntu Pro for Infrastructure: maintenance{% endblock %}
 
+{% block title %}{{ title | default("Ubuntu Pro for Infrastructure: maintenance") }}{% endblock %}
 
 {% block content %}
-<div class="p-strip">
-  <div class="row">
-    <div class="col-6 col-start-large-4">
-      <div class="p-card--outline">
-        <div class="row u-equal-height">
-          <div class="u-vertically-center">
-            <img src="https://assets.ubuntu.com/v1/05dad221-info.svg" alt="" style="width: 70px;">
-          </div>
-          <div class="u-vertically-center">
-            <h1 class="p-heading--3">We'll be back soon</h1>
-            <p class="p-heading--4">We're updating the Ubuntu Pro store</p>
+  <div class="p-strip">
+    <div class="row">
+      <div class="col-6 col-start-large-4">
+        <div class="p-card--outline">
+          <div class="row u-equal-height">
+            <div class="u-vertically-center">
+              <img src="https://assets.ubuntu.com/v1/05dad221-info.svg"
+                   alt=""
+                   style="width: 70px" />
```

Comment:
Use image template for here
```
{{ image (
          url="",
          alt="",
          width="",
          height="",
          hi_def=True,
          loading="lazy",
          attrs={"style": ""},
          ) | safe
        }}
```

Reply:
as far as I know across u.com, image templates are used for all images. Maybe for better consistency?

Reply:
This adds lazy loading and image quality to the images